### PR TITLE
Added travis CI support

### LINKED ISSUE #3: Add Travis CI support
# Description
Update the project with a proper travis ci file to run the tests (this is a good place to also check several Node.js versions so we can track when it breaks)

## Requirements
The project already has integration with Travis CI but it's missing the proper travis file with the right definition:
1. Node.js project
2. Test the following versions: 4, 6, and 8.

### PR DESCRIPTION
**Pull request by civan**
Added travis CI support for node versions: 4, 6 and 8.
Fixes #3 
take a look here: https://travis-ci.org/civan/licensewatch

### CODE REVIEW

**Comment by lirantal** [comment]
Thanks!
Can you also apply a fix to remove the node engines section from package.json now that the project supports it?

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "bluebird": "^3.4.7",
-    "glob": "^7.1.1"
+    "glob": "^7.1.1",
+    "licensewatch": "^1.2.1"
```
Comment (lirantal) [inline package.json]: why is this dependency added?

File: dist/index.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ var LicenseWatch = function (_EventEmitter) {
   _createClass(LicenseWatch, [{
     key: 'fetch',
     value: function () {
-      var _ref = _asyncToGenerator( /*#__PURE__*/regeneratorRuntime.mark(function _callee() {
+      var _ref = _asyncToGenerator(regeneratorRuntime.mark(function _callee() {
```
Comment (lirantal) [inline dist/index.js]: why this change happens?

**Review by lirantal — CHANGES_REQUESTED** [review]

File: dist/index.js (hunk 2 above)
Comment (civan) [reply dist/index.js]: probably something with the version. Question is shouldn't dist/ folder be ignored?

File: package.json (hunk 1 above)
Comment (civan) [reply package.json]: reverted

File: dist/index.js (hunk 2 above)
Comment (lirantal) [reply dist/index.js]: no, because we chose to ship it with the transpiled version rather than do it on the user's end install.

**Review by lirantal — APPROVED** [review]

**Comment by lirantal** [comment]
Great stuff, thanks!
